spec/lib/savings_transformation_spec.rb

```diff
@@ -51,7 +63,8 @@
 
     it 'sets passed to equal savings_and_investments_valid? on the application' do
       Saving.all.each do |saving|
```

Comment:
I wouldn't do the loop here, which uses the `savings_investment_valid?` method, which is 100% the same as the implementation, because that doesn't really test the result. I think it would be better to remove the method from the spec and here to test explicitly each case:

```ruby
  expect(application1.savings.passed).to be true
  expect(application2.savings.passed).to be false
  expect(application3.savings.passed).to be false
  expect(application4.savings.passed).to be true
  expect(application5.savings.passed).to be true
```

Or something like this. It's only 5 applications, so it's fine.